src/Scaffold_Command.php

```diff
@@ -366,7 +362,7 @@ function child_theme( $args, $assoc_args ) {
 		$theme_slug = $args[0];
 
 		if ( in_array( $theme_slug, array( '.', '..' ) ) ) {
-			WP_CLI::error( "Invalid theme slug specified." );
+			WP_CLI::error( "Invalid theme slug specified. The slug cannot be \".\" or \"..\"." );
```

Comment:
One minor nit. Is there a reason you went with double quotes instead of single quotes? If I recall correctly, we use single quotes pretty consistently elsewhere.